Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/SFTPLogger.cs

```diff
@@ -0,0 +1,293 @@
+﻿using System.Collections.Concurrent;
+using Serilog;
+
+namespace Frends.SFTP.UploadFiles.Definitions
+{
+    /// <summary>
+    /// SFTP internal logger interface
+    /// </summary>
+    public interface ISFTPLogger : IDisposable
```

Comment:
This can be internal and as such most comments are not necessary as they just state the obvious